### DISCUSSION TITLE
Redstone MoreRed support

### DISCUSSION THREAD

**Discussion by Crayonan** [MikaylaFischler/cc-mek-scada#414] [Support]
Hi!

Let's start by saying hats of for developing this awesome project. At the time of writing Immersive Engineering is only available for NeoForge 1.20.1, I have my modpack which uses **[MoreRed](https://www.curseforge.com/minecraft/mc-mods/more-red)** with the **[CC: Tweaked compatibility mod ](https://www.curseforge.com/minecraft/mc-mods/more-red-x-cc-tweaked-compat)**. How would I configure the redstone if I were to use this mod but also make use of the graphic installer. Currently it's not receiving the redstone properly, or I haven't configured it properly. Because Immersive engineering is not available for Forge 1.20.1 I'm curious if this could be the work around.  

SS of the items from More Red, but I included links to the mods.
![image](https://github.com/MikaylaFischler/cc-mek-scada/assets/78920025/794ea939-d0b7-497d-829e-91ad47fe4b21)

**Comment by Crayonan** [comment]
Immersive Engineering is compatible, my bad!

**Reply by MikaylaFischler** [reply]
No worries :)
